jekyll/_cci2/use-the-cli-and-vcs-for-config-policy-management.adoc

```diff
@@ -111,6 +144,26 @@ To illustrate making a change to an existing policy, suppose you made an error w
 circleci-cli policy push ./config-policies --owner-id <your-organization-ID>
 ----
 
+[#config-policy-management-enablement]
+== Enable/Disable Policy Evaluation in CI
+
+This option controls policy evaluation in CI build pipeline. The setting can be modified using `--enabled` flag.
+
+If --enabled is set to `true`: project configuration will be evaluated against your organization's policies in CI
+If --enabled is set to `false`: project configuration will *not* be evaluated against your organization's policies in CI
```

Comment:
```suggestion
* If `--enabled` is set to `true`, project configuration **will** be evaluated against your organization's policies when pipelines are triggeres
* If `--enabled` is set to `false`, project configuration will *not* be evaluated against your organization's policies when pipelines are triggered.
```

Reply:
*triggered

Reply:
**will** is formatted with double asterisks,
*not* with single